corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -66,7 +162,14 @@ var ReportModule = (function () {
 
         function newReport(options) {
             options = options || {};
-            var report = new ReportConfig(options.report_id, options.header, self.availableReportIds, self.lang);
+            var report = new ReportConfig(
+                options.report_id,
```

Comment:
This is not high-level feedback, and you didn't introduce it, but casing inconsistency is noticeably weird here, where options properties follow one convention and self's properties another.